ipaserver/plugins/trust.py

```diff
@@ -527,8 +508,7 @@ class trust(LDAPObject):
         Str('cn',
             cli_name='realm',
             label=_('Realm name'),
-            primary_key=True,
-        ),
+            primary_key=True),
```

Comment:
The former style is actually the preferred style

```
Foo(
    arg1,
    arg2,
    arg3,
),
Foo(
    ...
),
```

Reply:
Tell it to a linting code because this is what forced me to change it to.